# library
My list of publications and resources to read through

## Papers and Publications  

### single cell RNA-seq  

#### software/packages  
* Andreatta and Carmona et al 2021 [UCell: Robust and scalable single-cell gene signature scoring](https://www.sciencedirect.com/science/article/pii/S2001037021002816)  


#### methods review  
* Rich et al 2024 [The impact of package selection and versioning on single-cell RNA-seq analysis](https://www.biorxiv.org/content/10.1101/2024.04.04.588111v1) 
* Cuomo et al 2021 [Optimizing expression quantitative trait locus mapping workflows for single-cell studies](https://pubmed.ncbi.nlm.nih.gov/34167583/)  
* Luecken and Theis 2019 [Current best practices in single-cell RNA-seq
analysis: a tutorial](https://www.embopress.org/doi/full/10.15252/msb.20188746)  
* Squair et al 2021 [Confronting false discoveries in single-cell differential expression](https://www.nature.com/articles/s41467-021-25960-2)  
* Zimmerman, Espeland & Langefeld 2021 [A practical solution to pseudoreplication bias in single-cell studies](https://www.nature.com/articles/s41467-021-21038-1)  

#### clustering and dimensionality reduction   
* Becht et al 2019 [Dimensionality reduction for visualizing single-cell data using UMAP(https://www.nature.com/articles/nbt.4314)  
* Traag, Waltman & van Eck 2019 [From Louvain to Leiden: guaranteeing well-connected communities](https://www.nature.com/articles/s41598-019-41695-z)  

#### integration  
* Antonsson and Melsted 2024 [Batch correction methods used in single cell RNA-sequencing analyses are often poorly calibrated](https://www.biorxiv.org/content/10.1101/2024.03.19.585562v1)  
* Ma et al 2023 [Principled and interpretable alignability testing and integration of single-cell data](https://www.pnas.org/doi/10.1073/pnas.2313719121)  
* Chari and Pachter 2023 [The specious art of single-cell genomics](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011288)  

<br/>  

### statistics  
* Parab and Bhalerao 2010 [Choosing statistical test](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996580/)  
* Finak et al 2015 [MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996580/)  
* Dong et al 2021 [powerEQTL: an R package and shiny application for sample size and power calculation of bulk tissue and single-cell eQTL analysis](https://academic.oup.com/bioinformatics/article/37/22/4269/6278296)  
* Thiese, Ronna, and Ott 2016 [P value interpretations and considerations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059270/)  
* Serdar et al 2020 [Sample size, power and effect size revisited: simplified and practical approaches in pre-clinical, clinical and laboratory studies](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7745163/)    
* Szumilas 2020 [Explaining Odds Ratios](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2938757/)  
* Burgess et al 2023 [Guidelines for performing Mendelian randomization investigations: update for summer 2023](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7384151/)  
* Hartwig et al 2016 [Two-sample Mendelian randomization: avoiding the downsides of a powerful, widely applicable but potentially fallible technique](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5722032/)  
* Davies et al 2017 [Reading Mendelian randomisation studies: a guide, glossary, and checklist for clinicians](https://www.bmj.com/content/bmj/362/bmj.k601.full.pdf)  


### CRISPR sgRNA functional screens  
* Li et al 2014 [MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0554-4)  
* Spahn et al 2017 [PinAPL-Py: A comprehensive web-application for the analysis of CRISPR/Cas9 screens](https://www.nature.com/articles/s41598-017-16193-9)  
* Bland 2000 [The odds ratio](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1127651/)  

### ATAC-seq  
* Smith and Sheffield 2020 [Analytic Approaches for ATAC-seq Data Analysis](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8191135/)  
* Yan, Powell, Curtis, and Wong, 2020 [From reads to insight: a hitchhiker's guide to ATAC-seq data analysis](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-1929-3)  
* Grandi, Modi, Kampman and Corces 2022 [Chromatin accessibility profiling by ATAC-seq](https://www.nature.com/articles/s41596-022-00692-9)  
* Reske, Wilson, and Chandler 2020 [ATAC-seq normalization method can significantly affect differential accessibility analysis and interpretation](https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00342-y)  
* Klemm, Shipony, and Greenleaf 2019 [Chromatin accessibility and the regulatory epigenome](https://www.nature.com/articles/s41576-018-0089-8)  
* Tarbell and Liu 2019 [HMMRATAC: a Hidden Markov ModeleR for ATAC-seq](https://academic.oup.com/nar/article/47/16/e91/5519166)  


### single cell ATAC-seq  
* Fang et al 2021 [Comprehensive analysis of single cell ATAC-seq
data with SnapATAC](https://www.nature.com/articles/s41467-021-21583-9)  
* Granja et al 2021 [ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis](ArchR is a scalable software package for integrative single-cell chromatin accessibility analysis)  

## reproducible research (workflow management, containers, version control)  
* Molder et al 2021 [Sustainable data analysis with Snakemake](https://f1000research.com/articles/10-33/v2)  


## software, packages, resources, blogs  

### statistics  

#### R  
* [rstatix](https://rpkgs.datanovia.com/rstatix/)  
