# library
My list of publications and resources to read through

## Papers and Publications  

### single cell RNA-seq  


#### methods review  
* Rich et al 2024 [The impact of package selection and versioning on single-cell RNA-seq analysis](https://www.biorxiv.org/content/10.1101/2024.04.04.588111v1)  



#### integration  
* Antonsson and Melsted 2024 [Batch correction methods used in single cell RNA-sequencing analyses are often poorly calibrated](https://www.biorxiv.org/content/10.1101/2024.03.19.585562v1)  
* Ma et al 2023 [Principled and interpretable alignability testing and integration of single-cell data](https://www.pnas.org/doi/10.1073/pnas.2313719121)  
* Chari and Pachter 2023 [The specious art of single-cell genomics](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011288)  

<br/>  

### statistics  
* Parab and Bhalerao 2010 [Choosing statistical test](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996580/)  
* Finak et al 2015 [MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996580/)  
* Dong et al 2021 [powerEQTL: an R package and shiny application for sample size and power calculation of bulk tissue and single-cell eQTL analysis](https://academic.oup.com/bioinformatics/article/37/22/4269/6278296)  
* Thiese, Ronna, and Ott 2016 [P value interpretations and considerations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059270/)  
* Bland 2000 [The odds ratio](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1127651/)  

## reproducible research (workflow management, containers, version control)  
* Molder et al 2021 [Sustainable data analysis with Snakemake](https://f1000research.com/articles/10-33/v2)  


## software, packages, resources, blogs  

### statistics  

#### R  
* [rstatix](https://rpkgs.datanovia.com/rstatix/)  
