# library
My list of publications and resources to read through

## Papers and Publications  

### single cell RNA-seq  

#### integration  
* Antonsson and Melsted 2024 [Batch correction methods used in single cell RNA-sequencing analyses are often poorly calibrated](https://www.biorxiv.org/content/10.1101/2024.03.19.585562v1)  

<br/>  

### statistics  
* Parab and Bhalerao 2010 [Choosing statistical test](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996580/)  
* Finak et al 2015 [MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996580/)  
* Dong et al 2021 [powerEQTL: an R package and shiny application for sample size and power calculation of bulk tissue and single-cell eQTL analysis](https://academic.oup.com/bioinformatics/article/37/22/4269/6278296)  
* Thiese, Ronna, and Ott 2016 [P value interpretations and considerations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5059270/)  
* Serdar et al 2020 [Sample size, power and effect size revisited: simplified and practical approaches in pre-clinical, clinical and laboratory studies](Sample size, power and effect size revisited: simplified and practical approaches in pre-clinical, clinical and laboratory studies)  
* Szumilas 2020 [Explaining Odds Ratios](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2938757/)  

### CRISPR sgRNA functional screens  
* Li et al 2014 [MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0554-4)  
* Spahn et al 2017 [PinAPL-Py: A comprehensive web-application for the analysis of CRISPR/Cas9 screens](https://www.nature.com/articles/s41598-017-16193-9)  

## software, packages, resources, blogs  

### statistics  

#### R  
* [rstatix](https://rpkgs.datanovia.com/rstatix/)  
